lynx_description/urdf/wheel.urdf.xacro

```diff
@@ -0,0 +1,76 @@
+
+    <xacro:if value="${prefix == 'fl'}">
+      <xacro:property name="x" value="${wheel_separation_x/2.0}" />
+      <xacro:property name="y" value="${wheel_config['wheel_separation']/2.0}" />
+      <xacro:property name="inertia_y_offset" value="${wheel_config['inertia_y_offset']}" />
+      <xacro:property name="fdir" value="1 -1 0" />
+    </xacro:if>
+    <xacro:if value="${prefix == 'fr'}">
+      <xacro:property name="x" value="${wheel_separation_x/2.0}" />
+      <xacro:property name="y" value="-${wheel_config['wheel_separation']/2.0}" />
+      <xacro:property name="inertia_y_offset" value="${-wheel_config['inertia_y_offset']}" />
+      <xacro:property name="fdir" value="1 1 0" />
+    </xacro:if>
+    <xacro:if value="${prefix == 'rl'}">
+      <xacro:property name="x" value="-${wheel_separation_x/2.0}" />
+      <xacro:property name="y" value="${wheel_config['wheel_separation']/2.0}" />
+      <xacro:property name="inertia_y_offset" value="${wheel_config['inertia_y_offset']}" />
+      <xacro:property name="fdir" value="1 1 0" />
+    </xacro:if>
+    <xacro:if value="${prefix == 'rr'}">
+      <xacro:property name="x" value="-${wheel_separation_x/2.0}" />
+      <xacro:property name="y" value="-${wheel_config['wheel_separation']/2.0}" />
+      <xacro:property name="inertia_y_offset" value="${-wheel_config['inertia_y_offset']}" />
+      <xacro:property name="fdir" value="1 -1 0" />
+    </xacro:if>
+
+    <joint name="${prefix}_wheel_joint" type="continuous">
+      <origin xyz="${x} ${y} 0.0" rpy="0.0 0.0 0.0" />
+      <parent link="body_link" />
+      <child link="${prefix}_wheel_link" />
+      <axis xyz="0.0 1.0 0.0" />
+
+      <limit effort="64.5" velocity="10.958" />
+      <dynamics damping="0.01" friction="0.9" />
+    </joint>
+
+    <link name="${prefix}_wheel_link">
+      <visual>
+        <origin xyz="0.0 0.0 0.0" rpy="-${pi/2} 0.0 0.0" />
+        <geometry>
+          <!-- TODO cheange when models ready -->
+          <!-- <mesh
+          filename="package://${wheel_config['mesh_package']}/${wheel_config['folder_path']}/${prefix}_wheel.dae"
+          /> -->
+          <cylinder radius="${wheel_config['wheel_radius']}" length="${wheel_config['wheel_width']}" />
+        </geometry>
+      </visual>
+
+      <collision>
+        <origin xyz="0.0 0.0 0.0" rpy="-${pi/2} 0.0 0.0" />
+        <geometry>
+          <cylinder radius="${wheel_config['wheel_radius']}" length="${wheel_config['wheel_width']}" />
+        </geometry>
+      </collision>
+
+      <inertial>
+        <origin xyz="0.0 ${inertia_y_offset} 0.0" rpy="0.0 0.0 0.0" />
+        <mass value="${wheel_config['mass']}" />
+        <inertia
+          ixx="${wheel_config['inertia']['ixx']}"
+          ixy="0.0"
+          ixz="0.0"
+          iyy="${wheel_config['inertia']['iyy']}"
+          iyz="0.0"
+          izz="${wheel_config['inertia']['izz']}" />
+      </inertial>
+    </link>
+
```

Comment:
If we add gazebo.urdf I propose to add also gazebo friction to not forget about it later.

```suggestion

    <xacro:if value="${use_sim}">
      <xacro:if value="${wheel_config['mecanum']}">
        <gazebo reference="${prefix}_wheel_link" xmlns:ignition="http://ignitionrobotics.org/schema">
          <collision>
            <surface>
              <friction>
                <ode>
                  <mu>1.0</mu>
                  <mu2>0.0</mu2>
                  <fdir1 ignition:expressed_in="base_link">${fdir}</fdir1>
                </ode>
              </friction>
            </surface>
          </collision>
        </gazebo>
      </xacro:if>
    </xacro:if>

```